Docs/featureguide/mixed precision/amp.rst

```diff
@@ -6,139 +6,130 @@
 
-    An example of a Pareto list:
+- Bitwidth: The bit width to which the layer group was changed
+- Accuracy: The accuracy of the model
+- Relative bit-ops: The bit-ops relative to starting
 
-    .. image:: ../../images/pareto.png
-        :width: 900px
+An example of a Pareto list:
 
-    Bit-ops are computed as
+.. image:: ../../images/pareto.png
+    :width: 900px
 
-    :math:`Bit-ops = Mac(op) * Bitwidth(parameter) * Bitwidth(Activation)`
+Bit-ops are computed as:
 
-    The Pareto list can be used for plotting a Pareto curve. A Bokeh plot for Pareto curve is generated and saved in the results directory.
+:math:`Bitops = Mac(op) * Bitwidth(parameter) * Bitwidth(Activation)`
 
-    .. image:: ../../images/pareto_curve.png
-        :width: 900px
+The Pareto list can be used for plotting a Pareto curve. A plot of the Pareto curve is generated using Bokeh and saved in the results directory.
 
-.. note::
+.. image:: ../../images/pareto_curve.png
+    :width: 900px
 
-    A user can pass two different evaluation callbacks for phase 1 and phase 2. Since phase 1 is measuring sensitivity
-    of each quantizer group, we can pass a smaller representative dataset for phase 1 for evaluation, or even use an indirect measure
-    such as SQNR which can be computed faster than but correlates well with the real evaluation metric.
+You can pass two different evaluation callbacks for phase 1 and phase 2.
 
-It is recommended to use the complete dataset for evaluation in phase 2.
+Since phase 1 measures sensitivity of each quantizer group, it can use a smaller representative dataset for evaluation, or even use an indirect measure such as SQNR that correlates with the direct evaluation metric but can be computed faster.
 
-4) Reduce Bit-width Convert Op Overhead (Phase 3)
--------------------------------------------------
+We recommend that you use the complete dataset for evaluation in phase 2.
 
-Convert Ops are introduced in the mixed-precision model for transition between Ops that are assigned different activation
-bit-widths or data types (float vs int). These Convert Ops contribute to the inference time along with bit-operations of Ops.
-In this phase the algorithm derives a mixed-precision solution having less Convert Op overhead w.r.t. to original solution
-keeping the mixed-precision accuracy intact. The algorithm produces mixed-precision solutions for a range of alpha values
-(0.0, 0.2, 0.4, 0.6, 0.8, 1.0) where the alpha represents fraction of original Convert Op overhead allowed for respective solution.
+Phase 4: Reduce bit-width convert op overhead
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Use Cases
-=========
+Conversion operations (convert ops) are introduced in the mixed-precision model for transition between ops with different activation bit widths or data types (float vs int). Convert ops contribute to the inference time along with bit-operations of ops.
```

Comment:
I assume that 'convert op' is jargon for 'conversion operation'. Is that the case?